example/vectorAdd/src/vectorAdd.cpp

```diff
@@ -182,13 +183,23 @@ auto main()
         numElements));
 
     // Enqueue the kernel execution task
-    alpaka::queue::enqueue(queue, taskKernel);
+    {
+        const auto beginT = std::chrono::high_resolution_clock::now();
+        alpaka::queue::enqueue(queue, taskKernel);
+        const auto endT = std::chrono::high_resolution_clock::now();
+        std::cout << "Time for kernel execution: " << std::chrono::duration<double>(endT-beginT).count() << 's' << std::endl;
+    }
```

Comment:
This is only for the blocking queue. Which it is in the example, but a user may change that when playing around.

Reply:
Added `alpaka::wait::wait(queue)`, here and for the copy further down.